tsl/test/sql/compression_update_delete.sql

```diff
@@ -941,3 +974,27 @@ SELECT COUNT(*) FROM :COMPRESS_CHUNK_1 WHERE c4 > 5 AND _ts_meta_min_2 <= 5 and
 -- report true
 SELECT COUNT(*) = :total_rows - :total_affected_rows FROM sample_table;
 ROLLBACK;
+
```

Comment:
This test doesn't really confirm we are using the index, they should work without one in an identical way.

Maybe we can use some debug messaging to confirm we are actually using the index.

Would also like a test with and without index and confirm we are getting same results.

Reply:
Now i output some debug info. Check below:

```
UPDATE sample_table SET c3 = c3 + 0 WHERE c4 = 4 AND c1 >= 7;
LOG:  statement: UPDATE sample_table SET c3 = c3 + 0 WHERE c4 = 4 AND c1 >= 7;
DEBUG:  Number of compressed rows fetched from index: 10 
Number of compressed rows filtered by orderby columns 7
```